src/lint/rules/tests/common.rs

```diff
@@ -20,6 +43,18 @@ pub fn assert_snippet(source_code: &str, expected_errors: usize, rules: &Current
                "{:#?}", lint_errors);
 }
 
+pub fn robust_assert_snippet(source_code: &str, expected_errors: Vec<ExpectedDMLStyleError>, rules: &CurrentRules) {
+    let lint_errors = run_linter(source_code, rules);
+    assert!(lint_errors.is_ok());
```

Comment:
No need for an additional assert here, unwrap() will already panic if it is an Err value

Reply:
Removed